elephant/current_source_density.py

```diff
@@ -227,7 +227,7 @@ def generate_lfp(csd_profile, x_positions, y_positions=None, z_positions=None,
         1D : gauss_1d_dipole
         2D : large_source_2D and small_source_2D
         3D : gauss_3d_dipole
-    x_positions : np.ndarray
+    x_positions : np.ndarray # TODO: add dimensionality for x-y-z_positions
```

Comment:
I assume this TODO should be addressed in another future PR and not here?

Reply:
Thank you for the reminder, I should have made this TODO more explicit.

I've now updated the docstring to explicitly specify the expected shape of the inputs as a 2D column vector with dimensions 'N x 1 array.' In the example mentioned in the issue, an error occurred because a flat array (1D) was passed as input. 

One might consider to further discuss the expected input shape in a future PR or Issue (?)
For now, documenting the expected input is a reasonable solution ?

Reply:
Yup, sounds good!